src/hooks/useAccountId.ts

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState, useCallback } from "react";
+import { AccountId } from "@hashgraph/sdk";
+import { convertAccountIdToEVMWallet_RPC } from "@/services/util/helpers";
+
+type FormValues = { [key: string]: string };
+
+export function useAccountId<T>(values: T, columnName: string) {
+  const [hederaAccountIdError, setHederaAccountIdError] = useState(false);
+  const [hederaEVMAccount, setHederaEVMAccount] = useState<string>();
```

Comment:
i think if we set the default value to empty string, it will infer the typing automatically without specifying it explicitly. it will also eliminate the default case when the value is undefined, as we need only strings here

const [hederaEVMAccount, setHederaEVMAccount] = useState("");